### DISCUSSION TITLE
--get-group-file在文件名有空格的时候该如何使用

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#169] [问题求助] [answered]
![39b4e3072ed62ac2.png](https://user-images.githubusercontent.com/66856838/115851512-a3c2e400-a3ec-11eb-9bae-29c71e08090c.png)

**Comment by Tsuk1ko** [comment, marked as answer]
确实没考虑到空格的问题，下个版本改进
